app/javascript/components/widget-controller.jsx

```diff
@@ -97,6 +104,14 @@ export class WidgetController extends React.Component {
     }));
   }
 
+  pause() {
+    if (this.state.isPaused) {
+      this.setState({ isPaused: 0 });
```

Comment:
best practice is to use a call back

ie `this.setState(({isPaused}) => ({isPaused: !isPaused}))`